CONTRIBUTING.md

```diff
@@ -42,6 +42,23 @@ If the hook complains and intercepts the commit, you can run PHP-CS-Fixer manual
 
     vendor/bin/php-cs-fixer fix src
 
+We expect contributors to code according to good coding practices also for front-end code. The Mozilla Developer Network has a strong position in the JavaScript/ECMAScript developer community, so we follow their example and use the [Prettier tool](https://prettier.io/docs/en/index.html) to validate the code.
+
+[Guidelines for styling JavaScript code examples](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide/Code_style_guide/JavaScript#general_guidelines_for_javascript_code_examples)
+
+To test how the Prettier tool works and what the rules are, you can visit the online version [here](https://prettier.io/playground/).
+
+Prettier validates and fixes the code with a pre-commit hook run on git commit. If there are style errors in the code, Prettier automatically corrects them in the files containing the code. If there are syntax errors, Prettier does not fix them, but will show you the errors that must be corrected before the commit can be performed successfully.
+
+If Prettier works unexpectedly, please let us know in the Skosmos [Users group](https://groups.google.com/g/skosmos-users).
+
+#### How to deploy Prettier in your Skosmos installation
+
+Add the following line at the end of the pre-commit file you already edited earlier related to PHP-CS-Fixer:
+```
+npx prettier --config prettier.config.js --write *.vue *.js
```

Comment:
I think it would be better to just include this line in the pre-commit file above.

Reply:
Also, now there is no distinction between just checking the style compliance (without doing any changes) and actually making the changes. I think it would be better to do it like this:

1. The developer is instructed how to run Prettier to fix JS code style.
2. The pre-commit hook only checks compliance but does not change the code. If the style check fails, it should output a list of problems/fixes (for example a diff, like php-cs-fixer) and bail out with an error code.
3. There should be a GitHub Action that also verifies compliance, without changing code. (This can be left for another PR)

Reply:
It seems that Prettier in the pre-commit has to be replaced by ESLint, so I deleted this line and created a new one for ESLint (npx eslint --ext .js *.js *.vue).

The changes I have made take into account your comments, but the step three (GitHub Action) I will do separately.